src/Assets/TestProjects/KitchenSink/TestApp/TestApp.csproj

```diff
@@ -18,6 +18,7 @@
     <TieredCompilationQuickJit>true</TieredCompilationQuickJit>
     <TieredCompilationQuickJitForLoops>true</TieredCompilationQuickJitForLoops>
     <StartupHookSupport>false</StartupHookSupport>
+    <BuiltInComSupport>false</BuiltInComSupport>
```

Comment:
Is this change necessary? It doesn't look like you're using this project in any of your tests.